cmd/clusterctl/examples/vsphere/machines.yaml.template

```diff
@@ -22,27 +22,27 @@ items:
           disk_label: ""
```

Comment:
Can we remove the `machineVariables:` part from the example? Since we are not using that anymore.
Since we haven't changed the structure, we can simply leave the empty key `machineVariables` and remove the keys inside it.

Reply:
Not yet.  I want this to go in without removing functionality.  We can do another PR to remove all the terraform code.